### PR TITLE
[Android/Test] script to run unittest

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Update script to run the unittest on target device.

Note that, the testcases are android instrumented test which requires the target device.

Cmd to build and run test: ./build-android-lib.sh --run_unittest=yes

Signed-off-by: Jaeyun Jung <jy1210.jung@samsung.com>

### CODE REVIEW

File: api/android/build-android-lib.sh [hunk 1]
```diff
@@ -70,26 +76,27 @@ echo "GStreamer binaries: $GSTREAMER_ROOT_ANDROID"
...
 
 # Copy the files (native and java to build Android library) to build directory
-cp -r $NNSTREAMER_ROOT/api/android/* ./build_android_lib
+cp -r ./api/android/* ./build_android_lib
 
 # Get the prebuilt libraries and build-script
 svn --force export https://github.com/nnsuite/nnstreamer-android-resource/trunk/android_api ./build_android_lib
 
-pushd ./build_android_lib
+cd ./build_android_lib
```
Comment (myungjoo) [inline api/android/build-android-lib.sh]: Let's use ```pushd``` for maintainability. It's especially good if it's nested as we do here.

File: api/android/build-android-lib.sh [hunk 2]
```diff
@@ -127,17 +135,27 @@ if [[ -e $nnstreamer_android_api_lib ]]; then
...
+
+        zip -r nnstreamer-api-unittest-$today.zip api/build/reports
+        cp nnstreamer-api-unittest-$today.zip ../$result_directory
+    fi
 else
     echo "Failed to build NNStreamer library."
 fi
 
-popd
+cd ..
```
Comment (myungjoo) [inline api/android/build-android-lib.sh]: Let's use popd, making it nested.

File: api/android/build-android-lib.sh (hunk 1 above)
Comment (jaeyun-jung) [reply api/android/build-android-lib.sh]: roll back this change

File: api/android/build-android-lib.sh (hunk 2 above)
Comment (jaeyun-jung) [reply api/android/build-android-lib.sh]: roll back this change

**Pull request merged**